service/controller/resource/tccp/create_test.go

```diff
@@ -43,35 +42,21 @@ func Test_Controller_Resource_TCCP_Template_Render(t *testing.T) {
 		errorMatcher   func(error) bool
 	}{
 		{
-			name:           "case 0: basic test without encryption key, route53 enabled",
+			name:           "case 0: basic test, route53 enabled",
 			cr:             unittest.DefaultCluster(),
 			ctx:            unittest.DefaultContext(),
 			errorMatcher:   nil,
 			route53Enabled: true,
 		},
 		{
-			name:           "case 1: basic test with encryption key, route53 enabled",
```

Comment:
The golden file tests are cleaned up because the encryption key does not play a role anymore in the TCCP stack.